app/models/intake/gyr_intake.rb

```diff
@@ -392,6 +398,16 @@ def bank_account
     @bank_account ||= BankAccount.new(account_type: type, bank_name: bank_name, account_number: bank_account_number, routing_number: bank_routing_number)
   end
 
+  def duplicates
+    return Model.none unless hashed_primary_ssn.present?
```

Comment:
@bytheway875 oh I'm unfamiliar with `Model.none`-- why wouldn't we just return nil?

Reply:
the intent was to return an empty active record relation instead of nil in the hope that it might be easier to deal with a reponse of the same "type." Second guessing whether that's useful, so def down to have you change it to return nil